pitest-command-line/src/test/java/org/pitest/mutationtest/commandline/CommaAwareArgsProcessorTest.java

```diff
@@ -0,0 +1,59 @@
+package org.pitest.mutationtest.commandline;
+
+import joptsimple.ArgumentAcceptingOptionSpec;
+import joptsimple.OptionParser;
+import joptsimple.OptionSet;
+import org.junit.Assert;
+import org.junit.Test;
+import org.junit.runner.RunWith;
+import org.junit.runners.Parameterized;
+import org.pitest.mutationtest.config.ConfigOption;
+
+import java.util.Arrays;
+import java.util.Collection;
+import java.util.Collections;
+import java.util.List;
+
+@RunWith(Parameterized.class)
+public class CommaAwareArgsProcessorTest {
+
+    @Parameterized.Parameters
+    public static Collection<Object[]> data() {
+        return Arrays.asList(new Object[][] {
+                {  Arrays.asList("--unrelated", "a"), Collections.emptyList() },
+                {  Arrays.asList("--jvmArgs", "a"), Arrays.asList("a") },
+                {  Arrays.asList("--jvmArgs", "a,b,c,d"), Arrays.asList("a","b","c","d") },
+                {  Arrays.asList("--jvmArgs", "a;b,c;d"), Arrays.asList("a;b","c;d") },
+                {  Arrays.asList("--jvmArgs", "{a,b,c,d}"), Arrays.asList("a,b,c,d") },
+                {  Arrays.asList("--jvmArgs", "{a;b;c;d}"), Arrays.asList("{a;b;c;d}") }, // no separators so region markers will not be removed
+                {  Arrays.asList("--jvmArgs", "{a,b},{c,d},{e,f}"), Arrays.asList("a,b","c,d","e,f") },
+                {  Arrays.asList("--jvmArgs", "{a,b@c,d}"), Arrays.asList("a,b@c,d") }, // pre-existing '@' will not be replaced with ","
```

Comment:
Is it possible to quote the `{` usage inside the command?

Reply:
That hasn't been implemented. I agree it would be nice to have, but I think we can probably wait until someone encounters a scenario where it causes a problem.

As far as I'm aware '{' are valid (although I've not turned up anything to confirm/deny this with a quick google). Don't think I've ever encountered it in practice though.